src/chainparams.cpp

```diff
@@ -352,8 +352,7 @@ class CTestNetParams : public CChainParams {
         consensus.vUpgrades[Consensus::UPGRADE_HEARTWOOD].hashActivationBlock =
             uint256S("05688d8a0e9ff7c04f6f05e6d695dc5ab43b9c4803342d77ae360b2b27d2468e");
         consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nProtocolVersion = 170012;
-        consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight =
-            Consensus::NetworkUpgrade::1020500;
+        consensus.vUpgrades[Consensus::UPGRADE_CANOPY].nActivationHeight = 1029140;
```

Comment:
One week of blocks is `7 * 24 * 48 = 8064`, while the difference in this commit is `1029140 - 1020500 = 8640`. Typo?

In any case, let's round the activation to the nearest 500.

Reply:
updated activation height to `1028500`, which gives us a difference of `1028500 - 1020500 = 8000`.